R/data-dictionary-checks.R

```diff
@@ -0,0 +1,151 @@
+# Data dictionary
+read_data_dictionary <- function() {
+  dd_file <- "data/data-dictionary.csv"
+  if (!file.exists(dd_file)) {
+    # Quick fix for running tests from testthat folder. Will be a better way to do this,
+    # but going to wait for someone to tell me what that is rather than spending time on
+    # StackOverflow.
+    # Since realised that it could go in known variables, but part of me wants it here to
+    # make it clearer what needs moving over to the package for these checks.
```

Comment:
Makes sense, we'll probably want to export the dictionary as a dataset in the package anyway which will simplify calling it in